modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -111,6 +111,10 @@ def test_read_csv_empty_frame(self):
 
         df_equals(modin_df, pandas_df)
 
+    def test_read_csv_without_glob(self):
+        with pytest.raises(FileNotFoundError):
```

Comment:
Does the FileNotFoundError here also return an exception message, and would it be useful to assert that the message matches?

Also, it is possible to check that the warning message from csv_glob_dispatcher.py is shown?

Reply:
Yes, it has an error message, but it comes from `pandas` and can change over time, which will make this test not resistant to change. From this point of view, it is better not to check the message.

Yes, it's possible. I'll add the check.

Reply:
@jeffreykennethli ready for review

Reply:
Test lgtm!